### ISSUE TITLE
[Documentation] Comparative benchmarks

### ISSUE THREAD

**Issue by arthurmensch** [scikit-learn-contrib/lightning#111]
It would be great to provide scripts that compares various estimators on problems of different size. I reckon the most pedagogical way to do so would be to display learning curves on a train and test set.
